api/src/main/java/io/minio/messages/ListBucketResultV2.java

```diff
@@ -27,22 +31,44 @@
  */
 @Root(name = "ListBucketResult", strict = false)
 @Namespace(reference = "http://s3.amazonaws.com/doc/2006-03-01/")
-public class ListBucketResultV2 extends ListBucketResult {
+public class ListBucketResultV2 extends ListObjectsResult {
+  @Element(name = "KeyCount", required = false)
+  private int keyCount;
+
+  @Element(name = "StartAfter", required = false)
+  private String startAfter;
+
   @Element(name = "ContinuationToken", required = false)
   private String continuationToken;
 
   @Element(name = "NextContinuationToken", required = false)
   private String nextContinuationToken;
 
+  @ElementList(name = "Contents", inline = true, required = false)
+  private List<Contents> contents;
+
+  /** Returns key count. */
+  public int keyCount() {
+    return keyCount;
+  }
+
+  /** Returns start after. */
+  public String startAfter() {
+    return startAfter;
+  }
+
   /** Returns continuation token. */
-  @Override
   public String continuationToken() {
     return continuationToken;
   }
 
   /** Returns next continuation token. */
-  @Override
   public String nextContinuationToken() {
     return nextContinuationToken;
   }
+
+  /** Returns List of Items. */
+  public List<Item> contents() {
```

Comment:
Add the `@Override` annotation, and change the implementation to call `emptyIfNull()` if you accept my suggestion in `ListObjectsResult`